shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/components/organisms/trades/TradeFlow01AccountDetails.kt

```diff
@@ -0,0 +1,53 @@
+package network.bisq.mobile.presentation.ui.components.organisms.trades
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import cafe.adriel.lyricist.LocalStrings
+import network.bisq.mobile.presentation.ui.components.atoms.BisqButton
+import network.bisq.mobile.presentation.ui.components.atoms.BisqText
+import network.bisq.mobile.presentation.ui.components.atoms.BisqTextField
+import network.bisq.mobile.presentation.ui.theme.BisqUIConstants
+import network.bisq.mobile.presentation.ui.uicases.trades.ITradeFlowPresenter
+import org.koin.compose.koinInject
+
+@Composable
+fun TradeFlow01AccountDetails(
```

Comment:
-optional- I would put the number (which indicates its position in the flow) as a header comment instead of as part of the function name

Reply:
Ha. I know it's bad to add the order number there. But somehow I felt it neat and did this.
Fixing it.